vmc/constants.go

```diff
@@ -46,6 +46,10 @@ const (
 	StorageScaleUpPolicyType = "storage-scaleup"
 	RapidScaleUpPolicyType   = "rapid-scaleup"
 
+	// Microsoft licensing config actions
+	LicenseConfigEnabled  = "ENABLED"
```

Comment:
I have opened a bug with VMC API team for accepting enabled and disabled. Currently the API only allows capitalized values and throws a 400 bad request otherwise.